test/runtests.jl

```diff
@@ -364,3 +364,29 @@ end
     @test_throws AssertionError restrict(h, 10, Inf)
 end
 
+@testset "Overflow" begin
+    a = randn(10^5)
+    b = randn(10^5)
+    ϵ = 1e-6
+    edges = -3:0.5:3
+    aclip = clamp.(a,first(edges)+ϵ,last(edges)-ϵ)
```

Comment:
what does `+ϵ` do here? to make the resultant value not sitting on the bin edge? 
maybe:
```suggestion
    aclip = clamp.(a,nextfloat(first(edges)),prevfloat(last(edges)))
```

Reply:
Ah didn't know about prevfloat so I used -𝜀 :) Thanks. Will fix.